DEVELOPING.md

```diff
@@ -21,7 +21,13 @@ outlined under [Dependencies](#dependencies) prior to attempting to build from
 ```
 cd polynote-frontend
 npm install
-npm run dist
+npm run build
```

Comment:
Hmm, we run `sbt dist` later in this section - do we need `npm run dist` for that to work? 

Maybe we should add a note in the `Running with IntelliJ` section about using `npm run watch`?